src/components/data/TableCell.tsx

```diff
@@ -38,6 +38,13 @@ const TableCellNext = function TableCell({
           // on scroll with sticky headers.
           'text-left border-t border-b border-b-grey-5': isHeadCell,
           'border-none': !isHeadCell,
+          // Apply a very subtle bottom border to the last row in the table (not
+          // in the head). This can help delineate the end of data in tables
+          // with sparse row data. Only apply border if row is not selected.
+          // This uses Tailwind's nested-group syntax. See
+          // https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups
+          'group-last/unselected:border-b group-last/unselected:border-grey-2 group-last/unselected:border-dotted':
+            !isHeadCell,
```

Comment:
I feel that most of Tailwind's naming conventions are self-documenting, or at least intuitive once you learn about them. The nested-group syntax is a stark departure from this: I don't find it easy to human-parse at all, so I've left a documentation link here.

`group-last/unselected:border-b` translated into some sort of English: _When a parent element with the `group/unselected` class is the last sibling element with `group/unselected` applied_. 😅 

Or, put another way, this could be thought of as:

```
tr.group/unselected:last-child td {
  // border rules
}
```

If you ignore for a moment that `/` is an illegal character in a classname.